solr/contrib/ltr/src/java/org/apache/solr/ltr/ranking/FilterFeature.java

```diff
@@ -47,10 +45,9 @@ public void init(Map<String,Object> params)
         + " init is not supported ("+this+")");
   }
 
-  public FilterFeature(Feature in, Normalizer norm) {
+  public FilterFeature(Feature in) {
```

Comment:
Do we even need a filter feature anymore?  Its initial purpose was to combine the norm + featureStore feature to make a model feature.  Now the model can use the featureStore features directly since it keeps track of the norms on its own